docs/Getting Started/Install and Run/index.md

```diff
@@ -49,7 +49,9 @@ CMake is used to configure your SDL Core build before you compile the project, t
 #### Development/Debug Options
 |Option|Value(s)|Dependencies|Description|
 |:-----|:-------|:-----------|:----------|
-|ENABLE_LOG|**ON**/OFF|log4cxx (included in project)|Enable/Disable logging tool. Logs are stored in `<sdl_build_dir>/bin/SmartDeviceLinkCore.log` and can be configured by `log4cxx.properties` in the same folder.|
+|ENABLE_LOG|**ON**/OFF|log4cxx (included in project)/boost logger|Enable/Disable logging tool. Logs are stored in `<sdl_build_dir>/bin/SmartDeviceLinkCore.log`.|
+|LOGGER_NAME|**LOG4CXX**|log4cxx (included in project)|Build with the apache log4cxx logger. Log properties can be configured in `<sdl_build_dir>/bin/log4cxx.properties`|
+|LOGGER_NAME|**BOOST**|boost logger|Build with the boost logger library. Log properties can be configured in `<sdl_build_dir>/bin/boostlogconfig.ini`|
```

Comment:
should these be two columns? this option is different from the rest of the table because the dependencies change, but since the two options are mutually exclusive and one is default i think it should be on one line. Let me know what you think of this suggestion

```suggestion
|LOGGER_NAME|**LOG4CXX**/BOOST|log4cxx (included in project)/boost logger|Build with the apache log4cxx logger or the boost logger library. Log properties can be configured in `<sdl_build_dir>/bin/log4cxx.properties` or `<sdl_build_dir>/bin/boostlogconfig.ini`|
```

Reply:
Hey @iCollin, I was basing it off of the style used for the `EXTENDED_POLICY` flag, which has separate columns for each value. I'm fine with either style. Let me know what you think.

Also, I agree that the bold should be removed from BOOST since it seems bold is only used for the default value(which in this case is LOG4CXX). Thanks for catching that 👍

Reply:
Ah I see, that makes good sense. Yeah just removing the bold from BOOST is all then.

```suggestion
|LOGGER_NAME|**LOG4CXX**|log4cxx (included in project)|Build with the apache log4cxx logger. Log properties can be configured in `<sdl_build_dir>/bin/log4cxx.properties`|
|LOGGER_NAME|BOOST|boost logger|Build with the boost logger library. Log properties can be configured in `<sdl_build_dir>/bin/boostlogconfig.ini`|
```